ecosystem/web-wallet/src/background.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosAccount, AptosClient, Types } from 'aptos'
+import { devnetNodeUrl } from './constants'
+import { getAptosAccountState } from './utils/account'
+
+chrome.runtime.onMessageExternal.addListener(async function (request, _sender, sendResponse) {
+  const account = getAptosAccountState()
+  if (account === undefined) {
+    sendResponse({ error: 'No Accounts' })
+  } else if (request.method === 'getAccountAddress') {
```

Comment:
Let's add an enum for message type and use exhaustive checking so we know that all message types are handled. https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#union-exhaustiveness-checking

Reply:
I added the enum, there is still an inconsistency with adding it to inpage.js because that doesn't have access to src. But I hope I can fix that up some point this week. We can talk about it some more in our 1:1 this week.